README.md

```diff
@@ -1,15 +1,195 @@
+variable "graphdb_license_path" {
+  description = “Path to your license file” 
+  type        = string
+  default 	   = "$PATH/$TO/Your/License.license"
+
+
+}
+
+
+variable "tags" {
+  description = "Common tags for all resources"
+  type        = map(string)
+  default     = {}
+}
+
+
+variable "ami_id" {
+  description = "AMI id"
+  type        = string
+  default     = null
+}
+#from Module VM
+variable "instance_type" {
+  description = "EC2 instance type"
+  type        = string
+  default     = null
+}
+
+
+variable "graphdb_version" {
+  description = "GraphDB version"
+  type        = string
+  default     = null
+}
+```
+
+Ensure that you adjust the values of the variables to match your specific requirements.
+
+
+
+## Examples
+
+This repository includes usage examples in the `examples/` directory, such as:
+
+- [VPC with Multiple Availability Zones](./examples/vpc-with-multiple-az): Demonstrates creating a VPC with multiple Availability Zones and provisioning an AWS GraphDB cluster.
+
+## Modules
+
+#### [Backup](modules/backup/README.md)
+This Terraform module creates an Amazon S3 bucket for storing backups.
+
+#### [Config](modules/config/README.md)
+This Terraform module creates AWS Systems Manager (SSM) parameters for managing sensitive data and configuration settings used by GraphDB in an AWS environment.
+
+#### [DNS](modules/dns/README.md)
+This Terraform module creates a private hosted zone in Amazon Route 53 for DNS resolution in your Virtual Private Cloud (VPC).
+
+#### [IAM](modules/iam/README.md)
+This Terraform module creates an AWS Identity and Access Management (IAM) role and an instance profile that can be used for EC2 instances. It also supports the option to set a permissions boundary on the IAM role. 
+
+#### [Load Balancer](modules/load_balancer/README.md)
+This Terraform module sets up an AWS Elastic Load Balancer (Network Load Balancer) with optional TLS listeners. The module is designed to be flexible and customizable by accepting various input variables to tailor the NLB configuration to your specific requirements.
+
+#### [User Data](modules/user_data/README.md)
+This Terraform module configures an AWS EC2 instance for running GraphDB with various optional parameters and user-supplied userdata.
+
+#### [VM](modules/vm/README.md)
+This Terraform module configures IAM roles, EC2 instances, and associated security groups for running GraphDB with optional parameters and customization.
+
+
+## Contributing
+Check out the contributors guide [CONTRIBUTING.md](CONTRIBUTING.md).
 
-TBD
 
 ## License
 
 This code is released under the Apache 2.0 License. See [LICENSE](LICENSE) for more details.
 
-## Contributing
 
-Check out the contributors guide [CONTRIBUTING.md](CONTRIBUTING.md).
+This README provides more detailed information about the important variables and how to configure them for the Terraform AWS GraphDB module. Please ensure that you adapt the configuration to match your specific use case and requirements.
```

Comment:
I think this is lost here at the bottom, maybe move it somewhere in the configurations sections ?